elements/pfe-styles/test/pfe-styles_e2e.js

```diff
@@ -14,4 +14,17 @@ describe(element, () => {
   it("should compare to the baseline", () => {
     expect(browser.checkFullPageScreen(element)).toBeLessThan(1.25);
   });
+
+  before(() => {
```

Comment:
There shouldn't be 2 before methods in a test.  You can update the url in the it statement as well.  I also don't know that this pattern needs the pause in either test case.

Reply:
I am feeling the struggle here to know what to do to create correct tests in PFE at all. Do we have any docs I should be referencing or can you provide some suggestions? Thank you!

Reply:
Sure we can talk about where we can improve documentation.  The before call is more of a mocha convention than specific to PFE: https://mochajs.org/#hooks